rasa_nlu/training_data/util.py

```diff
@@ -24,3 +25,23 @@ def check_duplicate_synonym(entity_synonyms, text, syn, context_str=""):
     if text in entity_synonyms and entity_synonyms[text] != syn:
         logger.warning("Found inconsistent entity synonyms while {0}, overwriting {1}->{2}"
                        "with {1}->{2} during merge".format(context_str, text, entity_synonyms[text], syn))
+
+
+def generate_lookup_regex(file_path, print_data_size=True):
+    """creates a regex out of the contents of a lookup table file"""
+    lookup_elements = []
+    with open(file_path, 'r') as f:
+        for l in f.readlines():
+            new_elements = [e.strip() for e in l.split(',')]
```

Comment:
i'm actually wondering if maybe we should just do a file where you only provide one word/phrase per line, rather than allowing new lines and comma separated?

Reply:
The way it is now we can do either or a combination of both.  I can't think of a good argument for restricting that.  Figured that the user might have different delimiters and wanted to minimize the amount of pre-processing that needed to happen.

Reply:
I see the point of @twhughes, but I'd go for restricting it to newlines for the moment as well. If new use cases come up that require separation on other characters, we can still add it again.
The reason is that once we add it, we need to support it for the future, and I'd rather avoid needing to support features where we are unsure about the usefulness.

Reply:
alright @twhughes once you've made that change i'll give this PR another review

Reply:
@akelad ok I made the change to the code and docs. only accepts newline-separated lookups now (commas will be treated as part of the phrase)